src/daemon/rpmostreed-transaction-types.cxx

```diff
@@ -1279,11 +1279,62 @@ deploy_transaction_execute (RpmostreedTransaction *transaction,
 
   if (self->override_replace_local_pkgs)
     {
+      OstreeDeployment *merge_deployment =
```

Comment:
This is duplicating a lot of the work we're already doing in the `override_reset_pkgs` case above. Let's try instead to reuse that work directly.

So e.g. instead of `else if (self->override_reset_pkgs)`, it'd be `else if (self->override_reset_pkgs || self->override_replace_local_pkgs)`. And then in that block, you'd add a separate loop for `self->override_replace_local_pkgs`. Does that make sense?

Reply:
Yes, it does actually ! Thanks. 
I almost found where the error happened using journalctl and printf (coredump had also info but I wasn't able to find nothing significant), I also found this:
```
cosa-devsh rpm-ostree[1263]: 'vv' is not a valid GVariant format string
cosa-devsh rpm-ostree[1263]: g_variant_get_child: assertion 'valid_format_string (format_string, TRUE, child)' failed
``` 
Nevermind, I'm just saying it to comment the progress, I'm now going to make that change and see if the errors it persist.

Reply:
The only problem is that self->override_replace_local_pkgs is a GUnixFDList* and not a char** , and it cant be iterate likewise, I'm going to try and find the way to iterate through it.